easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.5-goolfc-2017b.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = '7.0.5'
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = {'name': 'goolfc', 'version': '2017b'}
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+sources = ['%%(namelower)s-9.0-linux-x64-v%s.tgz' % version[0:1]]
```

Comment:
@JackPerdue What if there's an update of `cuDNN` 7.x? Name of tarball will not change, but checksum will, so problems?

That's why we rename with full `version` after download...

Reply:
How many versions of 9.0x7.0 have they released?    I only see one at their site.  I also see a 71 but that wouldn't conflict with this.

Where did Stephane get the .15 part of cuDNN-7.0.5.15-goolfc-2017b.eb?

If the rename is needed then it should be duly noted in the easyconfig instead of leaving people wondering where it came from (see email link above).

BTW, still think this one is better because it replaces 3 identical libraries with appropriate symlinks.

Reply:
Looks like Stephane (or somebody) got it right in other 9x7 builds (they included a comment on the renaming).   The current cuDNN-7.0.5.15-goolfc-2017b.eb could use such a comment.  It also wouldn't to hurt to mention that the toolchain uses CUDA 9.0 (which isn't a comment needed in the dummy builds).

Anyway.... still think my symlinks make it better. :)